src/components/Player.jsx

```diff
@@ -48,9 +48,9 @@ class Player extends React.Component {
     return (
       <div>
         <div data-vjs-player>
-          <video
+          <video-js
             ref={node => (this.videoNode = node)}
-            className="video-js vjs-16-9"
+            class="video-js vjs-16-9"
```

Comment:
`class` will throw a DOM prop warning since we're using JSX. We should change it back to 'className' unless there's a reason for the update.

Reply:
I'll have to double check, there were a _lot_ of tweaks needed to get rid of various errors and warnings.

Reply:
At the very least, `className` is standard practice in React components. If there was a warning, maybe it's because our linter doesn't play well React/JSX?

Reply:
For some reason `className` ends up as a `classname` attribute

```html
<video-js classname="video-js vjs-16-9" playsinline="true" id="vjs_video_1960" class="video-js vjs-paused vjs-controls-enabled vjs-workinghover vjs-v7 vjs-user-active vjs_video_1960-dimensions vjs-theme-forest" tabindex="-1" translate="no" role="region" aria-label="Video Player" lang="en">
```